src/rpc/rawtransaction.cpp

```diff
@@ -922,7 +922,7 @@ UniValue sendrawtransaction(const JSONRPCRequest& request)
     const uint256& hashTx = tx->GetHash();
 
     CAmount nMaxRawTxFee = maxTxFee;
-    if (request.params.size() > 1 && request.params[1].get_bool())
+    if (!request.params[1].isNull() && request.params[1].get_bool())
```

Comment:
This is safer. It throws an error when the user passes in "True" as a string (rather than silently evaluating to false)